File: Freestyle/Webservice/WcfService/Models/DTO/FromDb/Rhymes/Extend/ChooseWriteSentence.cs
//ChooseAnswerAuthorActivite
namespace WcfService.Models.DTO.FromDb.Rhymes.Extend
{
    using System.Collections.Generic;
    using System.Runtime.Serialization;
    using WcfService.Models.DTO.FromDb.Rhymes.Bases;
    using WcfService.Models.DTO.FromDb.Rhymes.Helpers;

    [DataContract]
    public class ChooseWriteSentence : IncompletedViewRhyme
    {
        [DataMember(EmitDefaultValue = false)]
        public List<ReplyToAccepted> SuggestedReplies;
    }
}

File: Freestyle/Android/Freestyle/src/freestyle/data/dto/fromdb/rhymes/extend/ChooseWriteSentence.java
package freestyle.data.dto.fromdb.rhymes.extend;
import java.util.List;

import freestyle.data.dto.fromdb.rhymes.bases.IncompletedViewRhyme;
import freestyle.data.dto.fromdb.rhymes.helpers.ReplyToAccepted;

public class ChooseWriteSentence extends IncompletedViewRhyme{
	public List<ReplyToAccepted> suggestedReplies;
}
